library_generation/owlbot/templates/poms/cloud_pom.xml.j2

```diff
@@ -62,6 +62,10 @@
       <groupId>com.google.api</groupId>
       <artifactId>gax-httpjson</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Is this dependency a must have for most of the new modules? If yes, I think we can make it a `test` as grpc modules are only used by unit tests. See examples in [google-cloud-java](https://github.com/googleapis/google-cloud-java/blob/4982e78e416fec80baff99f0131815e2d4955acf/java-speech/google-cloud-speech/pom.xml#L108-L112)

Reply:
I think the test scope is manually added ([commit](https://github.com/googleapis/google-cloud-java/pull/9258/commits/7512e3a0a0221070f8cf7991775033267d80dac1)).

We can make this change through template but I suggest we do it after the migration, along with other improvements.

Reply:
I see it's been there https://github.com/googleapis/synthtool/blob/master/docker/owlbot/java/templates/poms/cloud_pom.xml.j2#L67 for few years in synthtool's template.

Reply:
Thanks @suztomo. 

@diegomarquezp do you remember deleting this dependency when you migrate the template from synthtool?